_posts/2017-05-10-april.md

```diff
@@ -0,0 +1,86 @@
+
+We wanted our pitch to tell the reader what Eve is, what makes it unique, and why one would use it over other languages. From the pitch, you can see we arrived at these traits
+
+**Modern** - It was important for us to note that Eve is a new take on programming, specifically on "relational languages". On its own, relational may connote SQL, which is not what we want to do. By claiming that we are "modern" we hope to tell the reader that Eve is an updated approach to things they may have seen in the past.
+
+**Relational language** - there were several candidates to describe what *kind* of language Eve is. Pattern matching, logic, declarative. We decided on "relational" because relations are a concept central to programming in Eve, and we wanted to highlight this fact up-front.
+
+**Data-driven** - data is also a central concept to Eve, so having this word in our pitch was important to us. Of course, data is important in all programs, but claiming the language is "data-driven" gets across the idea that the language paradigm is not imperative. 
+
+**Without the boilerplate** - our central thesis in developing Eve is that current approaches to programming are hampered by incidental complexity. A lot of this complexity is driven by simple boilerplate - code that stands between you and the inherent complexity of your programming task. Boilerplate is a pain point for many developers, something we all understand is frustrating, but that we deal with nonetheless. If you're not sure about a modern relational language for writing data-driven programs, hopefully you can understand the value of a  language without boilerplate.
+
+#### The Eve Homepage
+
+We're going to be revamping our various web properties (the website, blog, and docs) in the coming weeks. We started the process with the homepage, giving it a fresh coat of paint and some new content. The various links still relate back to v0.2 content, but we'll be updating those to a consistent look with refreshed content in the coming weeks.
+
+### Community
+
+#### Eve Around the World
+
+**Copenhagen, June TBD**
+
+Zubair Quraishi is again hosting an Eve meetup in Copenhagen, scheduled for sometime in June. We'll post another update when he's set a firm date.
+
+**San Francisco, May 31**
+
+We're going to be holding our first official Eve meetup at the Eve office in downtown San Francisco in May. If you are interested in attending or have any ideas of what you'd like to learn, drop me a line via [e-mail](mailto:corey@kodowa.com) or [Twitter](https://twitter.com/with_eve).
```

Comment:
i would restate the day in full format, eg, Wednesday, May 31st, and also include a time or at least say there will be a further announcement with more specific details